package.json

```diff
@@ -3,15 +3,23 @@
   "version": "4.0.0",
   "description": "Automatically generate etags for HTTP responses, for Fastify",
   "main": "index.js",
+  "types": "types/index.d.ts",
   "scripts": {
     "bench": "node benchmarks/run.js",
-    "test": "standard | snazzy && tsd && tap -J test/*.test.js && npm run typescript",
-    "typescript": "tsd"
+    "lint": "standard | snazzy",
+    "test": "npm run test:unit && npm run test:typescript",
+    "test:unit": "tap",
+    "test:typescript": "tsd"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/fastify/fastify-etag.git"
   },
+  "files": [
+    "index.js",
+    "types/index.d.ts",
+    "fnv1a.js"
+  ],
```

Comment:
Let's avoid this. It's just a source of bugs.

Reply:
Also... include the test folder in the package.

Reply:
I know we had this discussion regarding tests already. I would prefer not to decide which files should be in the npm package. So i would like to remove the files section in package.json.